samples/apache-others/challenge/files/chroot/node/app.js

```diff
@@ -0,0 +1,15 @@
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 8080;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end(req.url.split('').reverse().join(''));
+});
+
+server.listen(port, hostname, () => {
+  // Send something to STDOUT to signal we are ready to accept connections
+  console.log(`Server running at http://${hostname}:${port}/`);
```

Comment:
I don't know of a more elegant way to do this.

Perhaps we can just have socat with `retry=unlimited`, but that also seems weird